dev-tools/check-shell-scripts.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -eu
+
+IFS='
+'
+
+SHELL_SCRIPTS=$(find ./ -type f -not -path '*/vendor/*' -iname '*.sh')
+
+# shellcheck disable=SC2086
```

Comment:
is this comment do anything (some bashism magic) or could be removed?

Reply:
ok, now I got lost.
fixing ShellCheck without adding it to CI will make the violations to appear again soon

Reply:
I added ShellCheck to CI. You can see it on line 14 in this file.